resolwe_bio/serializers.py

```diff
@@ -15,4 +15,12 @@ class SampleSerializer(CollectionSerializer):
     collections = serializers.PrimaryKeyRelatedField(many=True, read_only=True)
 
     class Meta(CollectionSerializer.Meta):
+        fields = CollectionSerializer.Meta.fields + ('collections', 'presample')
+
+
+class PresampleSerializer(CollectionSerializer):
+    collections = serializers.PrimaryKeyRelatedField(many=True, read_only=True)
+
+    class Meta(CollectionSerializer.Meta):
         fields = CollectionSerializer.Meta.fields + ('collections',)
+        read_only_fields = CollectionSerializer.Meta.read_only_fields + ('presample',)
```

Comment:
How can a user set `presample=True` if read only? The idea is that user will confirm (submit/finalize) the annotations when he is happy with it. Three requirement should be satisfied to graduate a presample to sample:
- Annotation is valid
- Data is attached to the sample object
- User is OK with the annotation